mappings/com/mojang/blaze3d/platform/GlStateManager.mapping

```diff
@@ -352,6 +352,16 @@ CLASS com/mojang/blaze3d/platform/GlStateManager com/mojang/blaze3d/platform/GlS
 		ARG 1 strings
 	METHOD lambda$static$0 (I)Lcom/mojang/blaze3d/platform/GlStateManager$C_vgejkzlz;
 		ARG 0 id
+	METHOD setupGui3DDiffuseLighting (Lorg/joml/Vector3f;Lorg/joml/Vector3f;)V
```

Comment:
I'm not sure if we use `3D` or `3d`

Reply:
`3D` had been used in these files in 1.19 so I carried it over, I can change it if wanted.

Reply:
It's alright then